src/TriggerBinding/SqlTriggerListener.cs

```diff
@@ -57,6 +62,14 @@ public SqlTriggerListener(string connectionString, string tableName, string user
             this._executor = executor;
             this._logger = logger;
             this._listenerState = ListenerNotStarted;
+
+            using (var connection = new SqlConnection(connectionString))
+            {
+                connection.Open();
```

Comment:
Instead of opening a connection just to get the connection properties, can we add the connection properties to this._telemetryProps [here](https://github.com/Azure/azure-functions-sql-extension/blob/4a43881144f6f1b15451bb6821c2cbf2f747b4b4/src/TriggerBinding/SqlTriggerListener.cs#L110) after the connection is opened? It is okay if the first event (StartListenerStart) does not have have the connection properties.

Reply:
Makes sense. I will update the code.